dev/com.ibm.ws.logging.osgi/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -91,4 +91,4 @@ console.format.simple=Use the simple logging format.
 console.format.json=Use the JSON logging format.
 
 json.fields=JSON field names that are being replaced
-json.fields.desc=When logs are in JSON format, use this attribute to replace default field names with new field names. Configure the new field name by using the following format: defaultFieldName:newFieldName. For field names that are associated with logs of a specified source, use the following format: [source:]?defaultFieldName:newFieldName, where [source] is the source you want to specify (such as message, trace, or accessLog). To rename multiple fields, specify a comma-separated list of field name mappings.
\ No newline at end of file
+json.fields.desc=When logs are in JSON format, use this attribute to replace default field names with new field names. Use this attribute to omit fields as well. Configure the new field name by using the following format: defaultFieldName:newFieldName?. For field names that are associated with logs of a specified source, use the following format: [source:]?defaultFieldName:newFieldName?, where [source] is the source you want to specify (such as message, trace, or accessLog). Leave newFieldNameblank if you want to omit the field. To rename/omit multiple fields, specify a comma-separated list of field name mappings.
```

Comment:
`Use this attribute to omit fields` - does this mean omit the filed from the logs?

` Leave newFieldNameblank if you want to omit the field`

Does this mean you would use use the following format?
`defaultFieldName:`

Reply:
Hi @dmuelle,

1. `Use this attribute to omit fields` - yes, the field will be omitted from the logs

2. `Leave newFieldNameblank if you want to omit the field`

Does this mean you would use use the following format?
`defaultFieldName:`

Yes that's correct, that is the format being used.